git-semver.sh

```diff
@@ -172,17 +172,22 @@ version-parse-pre-release() {
 version-get() {
     local sort_args version version_pre_releases pre_release_id_count pre_release_id_index
     local tags=$(git tag)
+    local sorted_tags=$(
+        echo "$tags" |
+            grep -oP "^${VERSION_PREFIX}\K[0-9]+\.[0-9]+\.[0-9]+.*" |
```

Comment:
This breaks on macOS because the `P` option does not exist.
In macOS, I replaced it with this and it worked:
```
perl -nle "m/^${VERSION_PREFIX}\K[0-9]+\.[0-9]+\.[0-9]+.*/ && print"
```